### PR TITLE
Fixes bug in picking channels in epochsTFR object

### PR DESCRIPTION
**Pull request by hejtmy**
Epochs TFR as well as Average TFR object has channels on a -3 axis. Picking channels exited in a bug.

### CODE REVIEW

**Comment by agramfort** [comment]
Thx can you update a test ?

**Comment by hejtmy** [comment]
Can do after a conference :) This is a very quick fix I needed for my analyses.

**Comment by agramfort** [comment]
ok  I took over. See:

https://github.com/mne-tools/mne-python/commit/71d27eb39ed4d1e699fb2bf7af512d513af674dd

thx @hejtmy for the bug fix.

File: mne/channels/channels.py [hunk 1]
```diff
@@ -751,7 +751,7 @@ def _pick_drop_channels(self, idx):
 
         if self.preload:
             # All others (Evoked, Epochs, Raw) have chs axis=-2
-            axis = -3 if isinstance(self, AverageTFR) else -2
+            axis = -3 if isinstance(self, (AverageTFR, EpochsTFR)) else -2
```
Comment (Eric89GXL) [inline mne/channels/channels.py]: This probably should be `_BaseTFR`

**Pull request merged**